src/dashboard/components/files/FileActions.js

```diff
@@ -29,15 +37,46 @@ export default function FileOptions ({missing, id, filePath, openFile}) {
     return <DeleteConfirmModal name={name} onDelete={deleteFile} onCancel={() => setDeleting(false)}/>
   }
 
+  const renameFile = () => {
+    console.log('start function')
+    // first, i need to open the dialog to ask the user where to save the file
```

Comment:
and you can remove all the comments too. Nobody wants to see my brain doing it's thing in here